internal/util/util.go

```diff
@@ -114,8 +116,12 @@ func AppendProjectEnvToCmdEnv(envMap map[string]string, envList []string) []stri
 	return envList
 }
 
-// TODO: indent each line for the modules.
-func IndentString(content string, spaces int) string {
-	// TODO: implement me 
-	return "Testing indent string retrun"
+// IndentString will Add  x space char padding at the beginging of each line.
+func IndentString(content string, space int) string {
+	var result bytes.Buffer
```

Comment:
You can do it without a byte buffer by just appending to a string.

`result += fmt.Sprintf(...`